src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,52 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
+    
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    
     public void getStatistic(String fromFileName, String toFileName) {
-
+        String line;
+        StringBuilder stringBuilder = new StringBuilder();
+        int summaSuply = 0;
+        int summaBuy = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] string = line.split(",");
+                stringBuilder.append(string[0]).append(" ").append(string[1]).append("\n");
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
Create message for exception
https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#dont-ignore-exceptions